share/wake/lib/gcc_wake/gcc.wake

```diff
@@ -25,15 +25,50 @@ export def releaseLFlags = (Nil)
 export def staticCFlags =  ("-Wall", "-Wno-format-security", "-O2", "-flto", Nil)
 export def staticLFlags  = ("-flto", "-static", "-s", Nil)
 
+target pwd _ =
+    makeExecPlan ("pwd",) Nil
+    | setPlanLocalOnly True
```

Comment:
Is there a specific reason this needs LocalOnly?

Reply:
If it isn't the pwd will be from inside the workspace so it will always emit `/workspace` which isn't what we want here. I was hopeful that compile_commands.json would allow relative paths but the spec doesn't say one way or the other and it only gives absolute paths as an example so I needed an absolute path, otherwise `"."` would have worked